acid/acid.py

```diff
@@ -1,16 +1,19 @@
 """An XBlock checking container/block relationships for correctness."""
 
 import logging
-import pkg_resources
 import random
+
+import pkg_resources
 import webob
 from lazy import lazy
 from mako.lookup import TemplateLookup
-
 from xblock.core import XBlock, XBlockAside
-from xblock.fields import Scope, Dict
-from xblock.fragment import Fragment
-import six
+from xblock.fields import Dict, Scope
+
+try:
+    from web_fragments.fragment import Fragment
+except:
+    from xblock.fragment import Fragment  # For backward compatibility
```

Comment:
```suggestion
    from xblock.fragment import Fragment  # For backward compatibility with quince and earlier.
```
Be more specific.  For backwards compatibility with what?